src/app/components/popup-choose-the-time/popup-choose-the-time.component.ts

```diff
@@ -1,11 +1,24 @@
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { ModalWindowService } from '../modal-window/modal-window.service';
 
 @Component({
   selector: 'app-popup-choose-the-time',
   templateUrl: './popup-choose-the-time.component.html',
   styleUrls: ['./popup-choose-the-time.component.scss'],
 })
-export class PopupChooseTheTimeComponent {
+export class PopupChooseTheTimeComponent implements OnInit {
+  constructor(private modalWindowService: ModalWindowService) {}
+
+  ngOnInit(): void {
+    this.modalWindowService.visible.subscribe((result: boolean) => {
+      console.log(result);
```

Comment:
What do we expect to recieve here?

Reply:
if we are get "false" - window will be close